### PR TITLE
GST & IET doc. - Update/Correction.

### PR DESCRIPTION
**Pull request by jkottiku**

### CODE REVIEW

**Review by manoj-freyr — APPROVED** [review]
looks good

File: FEATURES.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The P2P Benchmark and Qualification Tool is designed to provide the list of all
...
 
 ## GPU Stress Test - GST module
-The GPU Stress Test runs various GEMM operations as workloads to stress the GPU FLOPS performance. GEMM operations include SGEMM, DGEMM and HGEMM (Single/Double/Half-precision General Matrix Multiplication) operations based on configured parameters. The duration of the test is configurable, both in terms of time (how long to run) and iterations (how many times to run).
+The GPU Stress Test runs various GEMM computations as workloads to stress the GPU FLOPS performance and check whether it meets the configured target GFLOPS. GEMM workloads shall be configured as either operation type or data type. GEMM based on operation types include SGEMM, DGEMM and HGEMM (Single/Double/Half-precision General Matrix Multiplication) - configured using operation parameter. GEMM based on data types include fp8, i8, fp16, bf16, fp32 and  tf32 (xf32) - configured using data type parameter. The duration of the test is configurable, both in terms of time (how long to run) and iterations (how many times to run).
```
Comment (peterjunpark) [inline FEATURES.md:37]:
```suggestion
The GPU Stress Test runs various GEMM computations as workloads to stress the GPU FLOPS performance and check whether it meets the configured target GFLOPS. GEMM workloads shall be configured as either operation type or data type. GEMM based on operation types include SGEMM, DGEMM and HGEMM (Single/Double/Half-precision General Matrix Multiplication) - configured using operation parameter. GEMM based on data types include `fp8`, `i8`, `fp16`, `bf16`, `fp32` and `tf32` (`xf32`) - configured using data type parameter. The duration of the test is configurable, both in terms of time (how long to run) and iterations (how many times to run).
```
code font for readability

File: FEATURES.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ The P2P Benchmark and Qualification Tool is designed to provide the list of all
...
 
 ## Input EDPp Test - IET module
 The Input EDPp Test runs GEMM workloads to stress the GPU power (i.e. TGP). This test is used to verify if the GPU is capable of handling max. power stress for a sustained period of time. Also checks whether GPU power reaches a set target power.
```
Comment (peterjunpark) [inline FEATURES.md:40]:
```suggestion
The Input EDPp Test runs GEMM workloads to stress the GPU power (that is, TGP). This test is used to verify if the GPU is capable of handling max. power stress for a sustained period of time. Also checks whether GPU power reaches a set target power.
```
Avoid latin abbreviations (https://developers.google.com/style/abbreviations#dont-use)

File: docs/features.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ The P2P Benchmark and Qualification Tool is designed to provide the list of all
...
 
 ## GPU Stress Test - GST module
-The GPU Stress Test runs various GEMM operations as workloads to stress the GPU FLOPS performance. GEMM operations include SGEMM, DGEMM and HGEMM (Single/Double/Half-precision General Matrix Multiplication) operations based on configured parameters. The duration of the test is configurable, both in terms of time (how long to run) and iterations (how many times to run).
+The GPU Stress Test runs various GEMM computations as workloads to stress the GPU FLOPS performance and check whether it meets the configured target GFLOPS. GEMM workloads shall be configured as either operation type or data type. GEMM based on operation types include SGEMM, DGEMM and HGEMM (Single/Double/Half-precision General Matrix Multiplication) - configured using operation parameter. GEMM based on data types include fp8, i8, fp16, bf16, fp32 and  tf32 (xf32) - configured using data type parameter. The duration of the test is configurable, both in terms of time (how long to run) and iterations (how many times to run).
```
Comment (peterjunpark) [inline docs/features.md:37]:
```suggestion
The GPU Stress Test runs various GEMM computations as workloads to stress the GPU FLOPS performance and check whether it meets the configured target GFLOPS. GEMM workloads shall be configured as either operation type or data type. GEMM based on operation types include SGEMM, DGEMM and HGEMM (Single/Double/Half-precision General Matrix Multiplication) - configured using operation parameter. GEMM based on data types include `fp8`, `i8`, `fp16`, `bf16`, `fp32` and  `tf32` (`xf32`) - configured using data type parameter. The duration of the test is configurable, both in terms of time (how long to run) and iterations (how many times to run).
```

File: docs/features.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ The P2P Benchmark and Qualification Tool is designed to provide the list of all
...
 
 ## Input EDPp Test - IET module
 The Input EDPp Test runs GEMM workloads to stress the GPU power (i.e. TGP). This test is used to verify if the GPU is capable of handling max. power stress for a sustained period of time. Also checks whether GPU power reaches a set target power.
```
Comment (peterjunpark) [inline docs/features.md:40]:
```suggestion
The Input EDPp Test runs GEMM workloads to stress the GPU power (that is, TGP). This test is used to verify if the GPU is capable of handling max. power stress for a sustained period of time. Also checks whether GPU power reaches a set target power.
```
Avoid latin abbreviations (https://developers.google.com/style/abbreviations#dont-use)

File: docs/ug1main.md [hunk 5]
```diff
@@ -123,12 +123,10 @@ Please see the web page “ROCm, a New Era in Open GPU Computing” to find out
...
-
-The test should be capable driving the power level equivalent to the rated TDP of the card, or levels below that. The tool must be capable of driving cards at TDP-50% to TDP-100%, in 10% incremental jumps. This should be controllable by the user.
+The GPU Stress Test runs various GEMM computations as workloads to stress the GPU FLOPS performance and check whether it meets the configured target GFLOPS. GEMM workloads shall be configured as either operation type or data type. GEMM based on operation types include SGEMM, DGEMM and HGEMM (Single/Double/Half-precision General Matrix Multiplication) - configured using operation parameter. GEMM based on data types include fp8, i8, fp16, bf16, fp32 and  tf32 (xf32) - configured using data type parameter. The duration of the test is configurable, both in terms of time (how long to run) and iterations (how many times to run).
```
Comment (peterjunpark) [inline docs/ug1main.md:126]:
```suggestion
The GPU Stress Test runs various GEMM computations as workloads to stress the GPU FLOPS performance and check whether it meets the configured target GFLOPS. GEMM workloads shall be configured as either operation type or data type. GEMM based on operation types include SGEMM, DGEMM and HGEMM (Single/Double/Half-precision General Matrix Multiplication) - configured using operation parameter. GEMM based on data types include `fp8`, `i8`, `fp16`, `bf16`, `fp32` and  `tf32` (`xf32`) - configured using data type parameter. The duration of the test is configurable, both in terms of time (how long to run) and iterations (how many times to run).
```

File: docs/ug1main.md [hunk 6]
```diff
@@ -123,12 +123,10 @@ Please see the web page “ROCm, a New Era in Open GPU Computing” to find out
...
 
 #### Input EDPp Test - IET module
-The Input EDPp Test generates EDP peak power on all input rails. This test is used to verify if the system PSU is capable of handling the worst case power spikes of the board.  Peak Current at defined period  =  1 minute moving average power.
+The Input EDPp Test runs GEMM workloads to stress the GPU power (i.e. TGP). This test is used to verify if the GPU is capable of handling max. power stress for a sustained period of time. Also checks whether GPU power reaches a set target power.
```
Comment (peterjunpark) [inline docs/ug1main.md:129]:
```suggestion
The Input EDPp Test runs GEMM workloads to stress the GPU power (that is, TGP). This test is used to verify if the GPU is capable of handling max. power stress for a sustained period of time. Also checks whether GPU power reaches a set target power.
```

File: docs/ug1main.md [hunk 7]
```diff
@@ -2441,15 +2439,16 @@ Please note that in link information results, some records could be marked with
...
+The GPU Stress Test drives and measures the specified GPU(s) performance (GFLOPS) -
+by means of large matrix multiplications using GEMM operation types based computations like
+SGEMM/DGEMM/HGEMM (Single/Double-precision/Half-precision General Matrix Multiplication)
+or GEMM data types based computations like fp8, i8, fp16, bf16, fp32 or tf32 (xf32) via blas
+libraries like rocBLAS or hipBLASLt. The GPU stress module may be configured so it does not
+copy the host source matrix array to the GPU before every matrix multiplication. This allows
+the GPU performance to not be capped by device to host bandwidth transfers. The module calculates
+the GFLOPS performance for configured GEMM computation and checks if it meets configured
+performance target. The test passes if it achieves the target performance GFLOPS number
+during the duration of the test else reported as fail.
```
Comment (peterjunpark) [inline docs/ug1main.md:2451]:
```suggestion
The GPU Stress Test drives and measures the specified GPU(s) performance (GFLOPS) -
by means of large matrix multiplications using GEMM operation types based computations like
SGEMM/DGEMM/HGEMM (Single/Double-precision/Half-precision General Matrix Multiplication)
or GEMM data types based computations like `fp8`, `i8`, `fp16`, `bf16`, `fp32` or `tf32` (`xf32`) via BLAS
libraries like rocBLAS or hipBLASLt. The GPU stress module may be configured so it does not
copy the host source matrix array to the GPU before every matrix multiplication. This allows
the GPU performance to not be capped by device to host bandwidth transfers. The module calculates
the GFLOPS performance for configured GEMM computation and checks if it meets configured
performance target. The test passes if it achieves the target performance GFLOPS number
during the duration of the test else reported as fail.
```

File: docs/ug1main.md [hunk 8]
```diff
@@ -2727,14 +2726,12 @@ GPUs.
...
-GPU(s) to a pre-configured power level in watts by gradually increasing the
-compute load on the GPUs until the desired power level is achieved. This
-verifies that the GPUs can sustain a power level for a reasonable amount of time
-without problems like thermal violations arising.\n
+GPU (i.e. TGP) for a sustained duration of time. This tool leverage GEMM workload
+to drive the compute load on the GPU and check whether the power consumed meets configured
+target power in watts. The GEMM compute workloads are also pre-configured. This verifies
+that the GPUs can sustain a power level for a reasonable amount of time without problems
+like thermal violations arising. The test passes if GPU power meets/crosses the
+target power during the duration of the test else reported as fail.
```
Comment (peterjunpark) [inline docs/ug1main.md:2734]:
```suggestion
GPU (that is, TGP) for a sustained duration of time. This tool leverage GEMM workload
to drive the compute load on the GPU and check whether the power consumed meets configured
target power in watts. The GEMM compute workloads are also pre-configured. This verifies
that the GPUs can sustain a power level for a reasonable amount of time without problems
like thermal violations arising. The test passes if GPU power meets or crosses the
target power during the duration of the test else reported as fail.
```

**Review by peterjunpark — APPROVED** [review]
Some nitpicks

**Pull request merged**